go/src/koding/db/models/groupdata.go

```diff
@@ -0,0 +1,8 @@
+package models
+
+// GroupData holds a group's private info.
+type GroupData struct {
+	ID   string `json:"_id"`
+	Slug string `json:"slug"`
+	Data *Data  `bson:"data,omitempty" json:"data,omitempty"`
```

Comment:
I think it could be just `Data Data`, no need for pointer.

```go
type GroupData struct {
 	ID   string `json:"_id"`
 	Slug string `json:"slug"`
 	Data Data  `bson:"data,omitempty" json:"data,omitempty"`
```